src/scripts/runDatabaseMigrations.ts

```
@@ -0,0 +1,335 @@
+// CLI interface
+async function main() {
+  const args = process.argv.slice(2);
+  const command = args[0] || 'migrate';
+  
+  const runner = new MigrationRunner();
+
+  try {
+    switch (command) {
+      case 'migrate':
+        const targetVersion = args[1] ? parseInt(args[1]) : undefined;
+        await runner.runMigrations(targetVersion);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)